backend/wmg/pipeline/expression_summary_and_cell_counts.py

```diff
@@ -20,10 +21,20 @@
 
 class CensusParameters:
     census_version = "latest"
-    value_filter = {
-        "homo_sapiens": f"is_primary_data == True and nnz >= {GENE_EXPRESSION_COUNT_MIN_THRESHOLD}",
-        "mus_musculus": f"is_primary_data == True and nnz >= {GENE_EXPRESSION_COUNT_MIN_THRESHOLD}",
-    }
+
+    def value_filter(organism: str) -> str:
+        organism_mapping = {
+            "homo_sapiens": f"is_primary_data == True and nnz >= {GENE_EXPRESSION_COUNT_MIN_THRESHOLD}",
+            "mus_musculus": f"is_primary_data == True and nnz >= {GENE_EXPRESSION_COUNT_MIN_THRESHOLD}",
+        }
+        value_filter = organism_mapping[organism]
+        if FeatureFlagService.is_enabled(FeatureFlagValues.SCHEMA_4):
+            # Filter out non-tissue tissues and system-level tissues
+            # TODO: Once cellxgene-census is updated to support tiledbsoma 1.5.0, we can update the tissue filter to use `not in`:
```

Comment:
per discussion [here](https://czi-sci.slack.com/archives/C048BN7K6U8/p1698946086617329), we can't use `not in` in a query here until a newer version of census is released that supports tiledbsoma 1.5.0. census team expects this will be released ~next week. i think it makes sense to just commit something that we know will work now, and then maybe update this later if we care enough to.